webpack-widget.js

```diff
@@ -46,54 +64,61 @@ const configFromPackage = () => {
 	}
 }
 
-// This is a base config for building legacy widgets
-// It will skip webpack's javascript functionality
-// to avoid having to make changes to the source code of those widgets
-// the config argument allows you to override some settings
-// you can update the return from this method to modify or alter
-// the base configuration
-const getLegacyWidgetBuildConfig = (config = {}) => {
-	// load and combine the config
-	let materiaConfig = configFromPackage()
-	cfg = Object.assign({}, defaultCfg, {cleanName:materiaConfig.cleanName}, config)
-	// set up source and destination paths
-	let srcPath = cfg.srcPath + path.sep
-	let outputPath = cfg.outputPath + path.sep
+// Provides a default config option
+const combineConfig = (extras = {}) => {
+	const rules = getDefaultRules()
+	const orderedRules = [
+		rules.loaderDoNothingToJs,
+		rules.loaderCompileCoffee,
+		rules.copyImages,
+		rules.loadHTMLAndReplaceMateriaScripts,
+		rules.loadAndPrefixCSS,
+		rules.loadAndPrefixSASS
+	]
+
+	const pkgConfig = configFromPackage()
+	const defaultCfg = {
+		cleanName: pkgConfig.cleanName,
+		copyList: getDefaultCopyList(),
+		entries: getDefaultEntries(),
+		moduleRules: orderedRules
+	}
+
+	return Object.assign({}, defaultCfg, extras)
+}
 
-	//standard list of directories/files we want to copy as-is into build
-	let copyList = [
-		//tack on any additional files to copy that were specified by the widget
-		...cfg.preCopy,
+const getDefaultCopyList = () => {
```

Comment:
New Feature: getDefaultCopyList can now be used to create a copy of the copyList used in getLegacyWidgetBuildConfig. This can be used to alter, remove, and add to the files that are copied into the build directory.